group_project_v2/project_api/__init__.py

```diff
@@ -1,7 +1,5 @@
-''' API calls with respect group projects'''
-
+""" API calls with respect group projects"""
```

Comment:
@e-kolpakov What does "with respect group projects" mean? Seems like there might be a word or two missing here.